src/test/java/net/bramp/ffmpeg/ReadmeTest.java

```diff
@@ -68,12 +69,12 @@ public void testGetMediaInformation() throws IOException {
 
     FFmpegFormat format = probeResult.getFormat();
     String line1 =
-        String.format("File: '%s' ; Format: '%s' ; Duration: %.3fs", format.filename,
+        String.format(Locale.US, "File: '%s' ; Format: '%s' ; Duration: %.3fs", format.filename,
```

Comment:
Can we just call Locale.setDefault before the test, to avoid cluttering up the example